sources/api_client.go

```diff
@@ -10,7 +10,7 @@ import (
 	"net/url"
 	"strings"
 
-	sapi "github.com/lindgrenj6/sources-api-client-go"
+	sourcesapi "github.com/lindgrenj6/sources-api-client-go"
```

Comment:
We should probably move this into the RedHatInsights org

Reply:
Yep! Thats the plan after everthing is done. Probably tomorrow.